src/core/dto/profile.dto.ts

```diff
@@ -0,0 +1,76 @@
+import "reflect-metadata";
+
+import { Length } from "class-validator";
+import { Field, InputType } from "type-graphql";
+
+@InputType()
+export class CreateProfileInput {
+  @Field()
+  @Length(1, 255)
+  fullName!: string;
+
+  @Field()
+  pronouns!: string;
+
+  @Field()
+  tdaGradYear!: number;
+
+  @Field()
+  currentLocation!: string;
+
+  @Field()
+  @Length(1, 500)
+  bio!: string;
+
+  /*
+  @Field(() => String, { nullable: true })
+  profilePic: string | null | undefined;
+
+  @Field(() => String, { nullable: true })
+  homeCountry: string | null | undefined;
+
+  @Field(() => String, { nullable: true })
+  nickname: string | null | undefined;
+
+  @Field(() => String, { nullable: true })
+  website: string | null | undefined;
+
+  @Field()
+  tdaGradYearBannerVisible!: boolean;
+  */
+}
+
+@InputType()
+export class UpdateProfileInput {
+  @Field({ nullable: true })
+  @Length(1, 255)
+  fullName?: string;
+
+  @Field(() => String, { nullable: true })
+  pronouns?: string;
+
+  @Field(() => Number, { nullable: true })
+  tdaGradYear?: number;
+
+  @Field(() => String, { nullable: true })
+  currentLocation?: string;
+
+  @Field(() => String, { nullable: true })
+  @Length(1, 500)
+  bio?: string;
+
+  @Field(() => String, { nullable: true })
```

Comment:
Just linking my post in our slack channel about the `type | null | undefined` issue:

Ok guys, so remember how I had said that we should delete those `node_modules` that appear in the `prisma/` folder? Well, I def lied. Next time we run migrations and that folder appears, we need to drag and drop the `@generated` folder into our `node_modules` folder. 
[@Ryan Gibbs](https://tech-fleet-workspace.slack.com/team/U04V07BDZB3)
 this should fix the `null | undefined | string` issue, I think because I was having us delete the files that the type-graphql prisma integration was creating. OOPS.
so lets run `npm run reset:prisma` and put those files into node_modules.
my bad guys.